login.py

```diff
@@ -0,0 +1,32 @@
+import hashlib
+from datetime import datetime, timedelta
+
+import jwt
+from flask import request, make_response, jsonify, Blueprint
+from werkzeug.exceptions import Unauthorized
+
+from tables import User
+
+login_blueprint = Blueprint('login', __name__, url_prefix='/login')
+
+
+def decode_token(token):
```

Comment:
@YanaRudyi decode_token functions sound like more util function that could be reused in different controllers. It is always a good idea to move such functions to some `common.utils` in case if it doesn't have inner dependencies in application code.